src/common/gui/base/colorPicker/ColorPickerView.ts

```diff
@@ -0,0 +1,373 @@
+import m, { Children, Component, Vnode } from "mithril"
+import { px, size } from "../../size.js"
+import { ExpanderButton, ExpanderPanel } from "../Expander.js"
+import stream from "mithril/stream"
+import { TextField } from "../TextField.js"
+import { lang } from "../../../misc/LanguageViewModel.js"
+import { hexToHSL, hslToHex, isValidCSSHexColor, MAX_HUE_ANGLE, normalizeHueAngle } from "../Color.js"
+import { ColorPickerModel } from "./ColorPickerModel.js"
+import { client } from "../../../misc/ClientDetector.js"
+import { theme } from "../../theme.js"
+
+const enum Sizes {
```

Comment:
this shouldn't be an enum, this is not multiple cases of a type. I'd make them constants if you need to.